File: C++_Programming/Practice/Program6.cpp
#include<iostream>
using namespace std;

float AddTwoNumber(float iNo1, float iNo2)
{
    float fAns = 0.0f;
    fAns = iNo1 + iNo2;
    return fAns;
}

int main()
{
    float fValue1 = 0.0f;                   // To Store First Input
    float fValue2 = 0.0f;                   // To Store Second Input
    float fResult = 0.0f;                   // To Store the Result

    cout<<"Enter First Number : ";
    cin>>fValue1;
    
    cout<<"Enter Second Number : ";
    cin>>fValue2;

    fResult = AddTwoNumber(fValue1,fValue2);

    cout<<"Addition is : "<<fResult;

    return 0;
}

File: Java_Programming/Practice/program6.java
import java.util.Scanner;

// Class containing arithmetic methods.
class Arithmatic
{
    public static int AddTwoNumber(int iNo1, int iNo2)
    {
        int iResult = 0;

        iResult = iNo1 + iNo2;
    
        return iResult;
    }
}

// Main class of the application.
class program6
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        // Variable Creation with default values
        int iValue1 = 0;
        int iValue2 = 0;
        int iResult = 0;

        System.out.println("Enter First Number : ");
        iValue1 = sobj.nextInt();

        System.out.println("Enter Second Number : ");
        iValue2 = sobj.nextInt();

        iResult = Arithmatic.AddTwoNumber(iValue1,iValue2);

        System.out.println("Addition is : " + iResult);

        sobj.close();

    }
}